code/reports/rmdchunks/load-data.Rmd

```diff
@@ -28,7 +28,7 @@ raw_truth <- load_truth(
 )
 
 # get anomalies
-anomalies <- read_csv(here("data-truth", "anomalies", "anomalies.csv"))
+anomalies <- read_csv(paste0(hub_repo, "/raw/main/data-truth/anomalies/anomalies.csv"))
```

Comment:
This works on windows as well since it's not an actual path but a URL (`hub_repo = "https://github.com/epiforecasts/covid19-forecast-hub-europe"`). I can still change it. I didn't do it because I was thinking I needed to add URL params but turns out it's not necessary.

Reply:
Of course. My bad.